<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtNetwork</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="SliderX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>4</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remoto:Configuración</string>
    </property>
   </widget>
   <widget class="QSlider" name="SliderY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArmed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>ARMAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShoot">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>DISPARAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInformationServerClient">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modo: Sin conexión</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditStringData">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnableSliderButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual: Activar Sliders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSerialPort">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DisplayX">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DisplayY">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInfo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usa ASDW para mover los sliders  </string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTime">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LabelTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPort">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LabelPort</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
